apps/benchmark/src/components/Select.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { makeStyles, shorthands } from '@griffel/react';
+
+interface SelectProps {
+  label: string;
+}
+
+const useStyles = makeStyles({
```

Comment:
Let's use plain CSS for these styles to avoid any effects on the benchmark results.

Reply:
Done in [c3375d4](https://github.com/microsoft/griffel/pull/132/commits/c3375d4fc77d4ab50458d45b9de0eb35e1180609)